test/src/test/java/jenkins/security/BasicHeaderProcessorTest.java

```diff
@@ -81,13 +81,39 @@ private void makeRequestAndFail(String userAndPass) throws IOException, SAXExcep
     }
 
     private void makeRequestWithAuthAndVerify(String userAndPass, String username) throws IOException, SAXException {
+        String authCode = "Basic "+Scrambler.scramble(userAndPass);
+        doMakeRequestWithAuthAndVerify(authCode, username);
+    }
+
+    private void doMakeRequestWithAuthAndVerify(String authCode, String expected) throws IOException, SAXException {
```

Comment:
doMake... sounds like a Stapler endpoint prefix, but it's does not matter much for a private method